spec/stud/interval_spec.rb

```diff
@@ -24,6 +35,59 @@
       end
     end
 
+    it 'should be able to interrupt an interval within same thread' do
+      counter = 0
+
+      Stud.interval(0.5) do
+        counter += 1
```

Comment:
good suggestion, will postpone to another PR. 
@jordansissel where did you say that "promote comment to new issue" hidden option was? ;)